integrationtesting/opendcs-tests/src/test/java/org/opendcs/regression_tests/DBImportExportTest.java

```diff
@@ -0,0 +1,124 @@
+
+        assertExitNullOrZero();
+
+        String clearDB = SystemStubs.tapSystemOut(
+                () -> exit.execute(() ->
+                        DbImport.main(
+                                args("-l", logFile.getAbsolutePath(), "-W", "-y",
+                                        TestResources.getResource(configuration,"DBImportExportTest/existing.xml"))
+                        )
+                )
+        );
+        assertExitNullOrZero();
+
+        List<String> platargs =  new ArrayList<>(Arrays.asList("-l", logFile.getAbsolutePath(), "-d3"));
+        switch(type) {
+            case "normal":
+                log.info("Loading platform, routing spec, etc with no extra flags.");
+                break;
+            case "platonly":
+                log.info("Loading platforms with platform only flag.");
+                platargs.add("-p");
+                break;
+            case "platover":
+                log.info("Loading platforms with platform only and overwrite flags.");
+                platargs.addAll(Arrays.asList("-W", "-y", "-p"));
+                break;
+        }
+        String newFile = TestResources.getResource(configuration,"DBImportExportTest/new.xml");
+        platargs.add(newFile);
+
+        String importOut = SystemStubs.tapSystemOut(
+                            () -> exit.execute(() ->
+                                    DbImport.main(
+                                            platargs.toArray(new String[0])
+                                    )
+                            )
+                        );
+
+        assertExitNullOrZero();
+
+        String exportOut = SystemStubs.tapSystemOut(
+                () -> exit.execute(() ->
+                        DbExport.main(
+                                args("-l", logFile.getAbsolutePath(), "-d3")
+                        )
+                )
+        );
+        assertExitNullOrZero();
+
+        //String cleanedOut = cleanString(exportOut);
+
+        //File goldenFile = new File(TestResources.getResource(configuration,expectedResultFile));
+        //String golden = cleanString(IOUtils.toString(goldenFile.toURI().toURL().openStream(), StandardCharsets.UTF_8));
+        // code to generate golden results
+        FileWriter exportXMLFile = new FileWriter(TestResources.getResource(configuration,type + "export.xml"));
+        IOUtils.write(exportOut, exportXMLFile);
+        IOUtils.closeQuietly(exportXMLFile);
+        //assertEquals(golden, cleanedOut, "Output Doesn't match expected data."); //how to handle modified times?!?
+
+        try
+        {
+            int existingplats = configuration.getDecodesDatabase().platformList.size();
+            int existingrs = configuration.getDecodesDatabase().routingSpecList.size();
+            assertEquals(plats, existingplats, "Expected number of platforms not found!");
+            assertEquals(rs, existingrs, "Expected number of data sources not found!");
+        } catch (Throwable e)
```

Comment:
try catch unneccessary here. 

If you are concerned about some sort of throwing situation use `assertDoesNotThrow`